c/openssl/s3_srvr.blast.01.i.cil-1.c

```diff
@@ -1074,6 +1074,25 @@ int main(void)
   s->ctx = malloc(sizeof(SSL_CTX));
   s->session = malloc(sizeof(SSL_SESSION));
   s->state = 8464;
+
+	s->info_callback = (void (*) ()) __VERIFIER_nondet_ulong();
+  s->options = __VERIFIER_nondet_ulong();
+  s->verify_mode = __VERIFIER_nondet_int();
+  (s->session)->peer = (struct x509_st*) __VERIFIER_nondet_ulong();
```

Comment:
The same here, `peer` is agian used only in comparisions to 0. I wonder whether it wouldn't be better just to replace the comparisions with `__VERIFIER_nondet_bool`?